blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipelineNode.java

```diff
@@ -69,6 +70,18 @@
     @Navigable
     public abstract BluePipelineStepContainer getSteps();
 
+    /**
+     * @return Gives tests in this node.
+     */
+    @Navigable
+    public abstract BlueTestResultContainer getTests();
+
+    /**
+     * @return Gives the test summary for this node.
+     */
+    @Exported(name = TEST_SUMMARY, inline = true, skipNull = true)
+    public abstract BlueTestSummary getTestSummary();
```

Comment:
See my comments at implementation. Returning test summary as part of each node could be expensive.

Reply:
I don't think this is necessary for how it is implemented today so this method could be removed before merging. WDYT @abayer @vivek ?

Reply:
I honestly can't remember what the reason for this was. =)

Reply:
👍 yeah better to remove it from here.